src/components/EventListCard/EventListCard.tsx

```diff
@@ -53,6 +55,20 @@ function eventListCard(props: InterfaceEventListCardProps): JSX.Element {
     setEventModalIsOpen(false);
   };
 
+  const showDeleteModal = (): void => {
+    setEventDeleteModalIsOpen(true);
+  };
+  const hideDeleteModal = (): void => {
+    setEventDeleteModalIsOpen(false);
+  };
+
+  const showUpdateModel = (): void => {
```

Comment:
create a single toggleUpdateModal function instead of showUpdateModal and hideUpdateModal